style.css

```diff
@@ -496,3 +501,605 @@ button:active {
+  }
+
+  .social-media1 ul li a {
+    /* list-style: none; */
+    color: #505f79;
+  }
+
+  .resume {
+    width: fit-content;
+    background-color: #fff;
+  }
+
+  .languages {
+    display: flex;
+    flex-direction: column;
+    grid-column: auto;
+    padding-top: 24px;
+    padding-left: 0;
+  }
+
+  .lang {
+    display: flex;
+    height: 72px;
+    width: 100%;
+    justify-content: space-between;
+  }
+
+  .frame {
+    display: flex;
+    width: 100%;
+    justify-content: space-between;
+    border-bottom: 1px solid #dfe1e6;
+    padding-bottom: 20px;
+    padding-top: 20px;
+  }
+
+  .skil {
+    display: flex;
+    width: 684px;
+    flex-grow: 1;
+    justify-content: space-between;
+    border-bottom: 1px solid #dfe1e6;
+    padding-bottom: 20px;
+    padding-top: 20px;
+  }
+
+  .language-type img {
+    width: 100%;
+    height: 48px;
+  }
+
+  .language-type {
+    display: flex;
+    flex-direction: column;
+    background-color: #f7f7f9;
+    border-radius: 8px;
+    color: #253858;
+    width: 414px;
+    height: 120px;
+    font-weight: 500;
+    font-size: 15px;
+    line-height: 20px;
+    margin: 10px;
+    align-items: center;
+    padding: 12px;
+  }
+
+  .language-type p {
+    justify-content: center;
+    text-align: center;
+  }
+
+  /* contact-form */
+
+  .contact-form-container {
+    background-color: #6070ff;
+    width: 100%;
+    border-radius: 100px 0 0 0;
+    padding-bottom: 8px;
+    height: 846px;
+    margin-top: -256px;
+  }
+
+  .contact-form {
+    box-sizing: border-box;
+    display: flex;
+    flex-direction: column;
+    width: 100%;
+    justify-content: center;
+    background-image: url("images/contact-form-bg-desktop.png");
+    background-repeat: no-repeat;
+    background-size: cover;
+
+    /* background-position-x: 153px;
+    background-position-y: 21px; */
+  }
+
+  .contact-form h4 {
+    height: 56px;
+    width: 801px;
+    margin: 103px 0 0 0;
+    font-size: 40px;
+    font-style: normal;
+    font-weight: 700;
+    line-height: 52px;
+    text-align: center;
+    color: #fff;
+    padding-left: 320px;
+  }
+
+  .contact-form p {
+    height: 102px;
+    width: 569px;
+    margin: 12px 24px 69px 24px;
+    font-size: 20px;
+    font-style: normal;
+    font-weight: 400;
+    line-height: 28px;
+    text-align: center;
+    color: #ebebff;
+    margin-left: 436px;
+  }
+
+  .form-input {
+    height: 16px;
+    width: 447px;
+    font-size: 17px;
+    font-style: normal;
+    font-weight: 400;
+    line-height: 20px;
+    text-align: left;
+    color: #172b4d;
+    margin: 0 24px 24px 24px;
+    padding: 15px 38px 15px 16px;
+    border-radius: 8px;
+    border: 1px solid #cfd8dc;
+    margin-left: 497px;
+  }
+
+  #message {
+    height: 170px;
+    width: 447px;
+    margin-left: 497px;
+    margin-right: 496px;
+  }
+
+  #touch {
+    margin: 0 211px 0 24px;
+    width: 140px;
+    margin-left: 650.5px;
+  }
+
+  .form-bottom-line-shape {
+    text-align: center;
+  }
```

Comment:
- [ ] Please hide this line for the desktop version. :desktop_computer: 

> - **Your implementation:**
![l1](https://user-images.githubusercontent.com/70483835/154793004-2b8b67a5-4f55-49ca-bcde-0b5ebc81cdd0.png)
> - **Figma design**
![l2](https://user-images.githubusercontent.com/70483835/154793007-2d7c76ac-9911-4881-a8d4-380ab3c49753.png)